### PR TITLE
docs: adding session docs

### PR DESCRIPTION
**Pull request by tatang26**
This PR adds the session docs for Leapkit.

### CODE REVIEW

File: docs/core/session.md [hunk 1]
```diff
@@ -2,3 +2,53 @@
 index: 2
 title: "Session"
 ---
+
+Leapkit provides a way to set a session in your project to persist values needed throughout the project. Leapkit uses the [gorilla/sessions](https://github.com/gorilla/sessions) package for this.
+
+## Setup
+
+To set the session in your project, you have to use the `server.WithSession` server option and pass a session secret key and the session name.
```
Comment (paganotoni) [inline docs/core/session.md:10]: Your app. not project.

File: docs/core/session.md [hunk 2]
```diff
@@ -2,3 +2,53 @@
...
+
+    // getting a flash messages
+    flashes := ss.Flashes("username_flash")
+    // ["peter"]
+    // ...
+
+    // Saving the session
+    if err := ss.Save(r, w); err != nil {
+        // handle the error
+    }
```
Comment (paganotoni) [inline docs/core/session.md:50]: This part looks confusing to me @tatang26, can we avoid the Save?

File: docs/core/session.md (hunk 1 above)
Comment (tatang26) [reply docs/core/session.md:10]: Sure, I'll make the change

File: docs/core/session.md (hunk 2 above)
Comment (tatang26) [reply docs/core/session.md:50]: Sure, I'll remove it

**Pull request merged**